src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -20,20 +18,18 @@ <h2 class="slds-text-heading_medium slds-text-heading--medium slds-hyphenate hea
                     </div>
                 </div>
             </template>
-            <template if:true={isFieldInfoReady}>
+            <template if:true={isRecordReady}>
 
                 <div class={cssEditForm}>
-                    <template if:true={hasError}> 
-                        <c-util-page-level-message title={errorMessage.header}
-                                                subtitle={errorMessage.detail}
-                                                subtitle-class-override="slds-p-horizontal_medium slds-p-vertical_xx-small"
-                                                variant="error">
+                    <template if:true={hasError}>
+                        <c-util-page-level-message title={errorMessage.header} subtitle={errorMessage.detail}
+                            subtitle-class-override="slds-p-horizontal_medium slds-p-vertical_xx-small" variant="error">
                         </c-util-page-level-message>
                     </template>
-                
+
                     <div class="slds-section slds-is-open">
                         <h3 class="slds-section__title slds-theme_shade">
-                            <span class="slds-truncate slds-p-horizontal_small" title="Section Title">{customLabels.donorSectionHeader}</span>
+                            <span class="slds-truncate slds-p-horizontal_small">{customLabels.donorSectionHeader}</span>
```

Comment:
Removed title since it is not required and has text that is not going to be translated.